lib/overrides.js

```diff
@@ -88,18 +84,13 @@ module.exports = class Overrides {
   }
 
   static printList(list, indent = '') {
-    let output = '';
-
-    for (let item of list) {
-      if (Array.isArray(item)) {
-        output += `${indent}* ${item[0]}\n`;
-        output += Overrides.printList(item[1], indent + '  ');
-      } else {
-        output += `${indent}* ${item}\n`;
+    return list.reduce((listString, listItem) => {
+      if (Array.isArray(listItem)) {
+        listString += `${indent}* ${listItem[0]}\n`;
+        listString += Overrides.printList(listItem[1], indent + '  ');
       }
-    }
-
-    return output;
+      listString += `${indent}* ${listItem}\n`;
+    }, '');
```

Comment:
While I personally enjoy a good `reduce`, I think we're better leaving this code as is—if it's not specifically fixing a bug, we're just changing good working code for aesthetic reasons.